ts/navigation/AuthenticationNavigator.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import CardSelectionScreen from "../screens/authentication/CardSelectionScreen";
+import CieAuthorizeDataUsageScreen from "../screens/authentication/cie/CieAuthorizeDataUsageScreen";
+import CieCardReaderScreen from "../screens/authentication/cie/CieCardReaderScreen";
+import CieConsentDataUsageScreen from "../screens/authentication/cie/CieConsentDataUsageScreen";
+import CieExpiredOrInvalidScreen from "../screens/authentication/cie/CieExpiredOrInvalidScreen";
+import CiePinLockedTemporarilyScreen from "../screens/authentication/cie/CiePinLockedTemporarilyScreen";
+import CiePinScreen from "../screens/authentication/cie/CiePinScreen";
+import CieWrongCiePinScreen from "../screens/authentication/cie/CieWrongCiePinScreen";
+import IdpLoginScreen from "../screens/authentication/IdpLoginScreen";
+import IdpSelectionScreen from "../screens/authentication/IdpSelectionScreen";
+import LandingScreen from "../screens/authentication/LandingScreen";
+import SpidCIEInformationScreen from "../screens/authentication/SpidCIEInformationScreen";
+import SpidInformationScreen from "../screens/authentication/SpidInformationScreen";
+import TestAuthenticationScreen from "../screens/authentication/TestAuthenticationScreen";
+import MarkdownScreen from "../screens/development/MarkdownScreen";
+import { AuthenticationParamsList } from "./params/AuthenticationParamsList";
+import ROUTES from "./routes";
+
+const Stack = createStackNavigator<AuthenticationParamsList>();
+
+const AuthenticationStackNavigator = () => (
+  <Stack.Navigator
+    initialRouteName={ROUTES.AUTHENTICATION_LANDING}
+    headerMode={"none"}
+    screenOptions={{ gestureEnabled: true }}
+  >
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_LANDING}
+      component={LandingScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_IDP_SELECTION}
+      component={IdpSelectionScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_CIE}
+      component={CardSelectionScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_IDP_LOGIN}
+      component={IdpLoginScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_IDP_TEST}
+      component={TestAuthenticationScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_SPID_INFORMATION}
+      component={SpidInformationScreen}
+    />
+
+    <Stack.Screen
+      name={ROUTES.AUTHENTICATION_SPID_CIE_INFORMATION}
+      component={SpidCIEInformationScreen}
```

Comment:
The same of the previous comments, also for `SpidCIEInformationScreen`. In addition, in this screen seems that the navigation prop is never used, maybe can be removed?